CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.3.1 (2019-08-13)
+
+- Adjusted CSS styles for `sky-code` component, reverted to previous color scheme. [#22](https://github.com/blackbaud/skyux-lib-code-block/pull/22)
```

Comment:
I'd add "the" before "sky-code," change the comma to "and," and add "the" before "previous."